File: _persist/temp/venda/CSharp/Empresa.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("EMPRESA")]
    public class Empresa : CollectionItem
    {
        [Campo("ID_EMPRESA", CampoTipo.tfKey)]
        public int Id_Empresa { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("ID_PESSOA", CampoTipo.tfReq)]
        public string Id_Pessoa { get; set; }
    }
}

File: _persist/temp/venda/Java/Empresa.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("EMPRESA")
public class Empresa extends CollectionItem
{
    @Campo("ID_EMPRESA", CampoTipo.tfKey)
    public Integer Id_Empresa;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("ID_PESSOA", CampoTipo.tfReq)
    public String Id_Pessoa;
}